core/src/main/scala/geotrellis/server/core/maml/package.scala

```diff
@@ -0,0 +1,24 @@
+package geotrellis.server.core
+
+import com.azavea.maml.ast.Expression
+import cats._
+import cats.effect.IO
+import io.circe._
+import com.googlecode.concurrentlinkedhashmap.ConcurrentLinkedHashMap
+
+import java.util.UUID
+
+
+package object maml {
+  type HashMapMamlStore = ConcurrentLinkedHashMap[UUID, Expression]
```

Comment:
Do we really need there `ConcurrentLinkedHashMap`?

Reply:
Perhaps not, but it makes demonstration very simple